packages/framework/src/schemas/steps/channels/in-app.schema.ts

```diff
@@ -1,10 +1,20 @@
 import { Schema } from '../../../types/schema.types';
 
+const redirectSchema = {
+  type: 'object',
+  properties: {
+    url: { type: 'string' },
+    target: { enum: ['_self', '_blank', '_parent', '_top', '_unfencedTop'] },
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Do we want to have a default target?‏

Reply:
🤔 I guess yes, `default: '_blank'`  will be appropriate